### DISCUSSION TITLE
Performance / WER overview of all model options

### DISCUSSION THREAD

**Discussion by Appfinity-development** [SYSTRAN/faster-whisper#1166] [General]
Is there a comprehensive overview that shows the difference between the various model option the faster whisper library has.

Eg `distil-large-v3` vs `large-v3` vs `large-v3-turbo`

I know performance also heavily depends on model configuration. But still some general recommendations would be great from the authors (@MahmoudAshraf97).

OpenAI did publish an overview when releasing their 'large-v3-turbo' in October:

https://github.com/openai/whisper/discussions/2363

But it still hard to decide what would give best results for your own use case.

**Comment by Purfview** [comment]
>what would give best results 

large-v2

**Comment by MahmoudAshraf97** [comment]
This is the only comparison I have that contain all models
https://github.com/SYSTRAN/faster-whisper/pull/1101#issue-2619272709
But you still have to benchmark on your own use case to find the best WER/speed tradeoff
